ddp-workspace/projects/ddp-sdk/src/lib/components/cookies/cookiesBanner/cookiesBanner.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Inject } from '@angular/core';
+import { CookiesManagementService } from '../../../services/cookiesManagement.service';
+import { MatDialog } from '@angular/material/dialog';
+import { ConfigurationService } from '../../../services/configuration.service';
+import { NoopScrollStrategy } from '@angular/cdk/overlay';
+import { CookiesPreferencesModalComponent } from '../cookiesPreferencesModal/cookiesPreferencesModal.component';
+import { PrivacyPolicyModalComponent } from '../privacyPolicy/privacyPolicyModal.component';
+
+@Component({
+  selector: 'ddp-cookies-banner',
+  template: `
+  <div class="cookiesBanner">
+    <div class="cookiesBanner__text">
```

Comment:
This is a picky comment, but maybe change stuff like "cookiesBanner__text" to "cookiesBanner--text" to better match the rest of the code.

Reply:
Done in b7b6914